### ISSUE TITLE
pingproxy is failing when running with python3

### ISSUE THREAD

**Issue by patrickelectric** [bluerobotics/ping-python#116]

**Comment by ES-Alexander** [comment]
Raised in [this forum post](https://discuss.bluerobotics.com/t/ping-echosounder-via-pingproxy-py-shown-in-pingviewer-but-no-output/11069)

I tried updating Companion 0 to use Python 3 for pingproxy, and it had the same behaviour as described (Ping Viewer detects the device, comes up as green, but then clicking into it yields no profiles).

I tried modifying pingproxy to [print when it writes to client](https://github.com/bluerobotics/ping-python/blob/master/tools/pingproxy.py#L87), and added a print statement to print the message when it [writes to the device](https://github.com/bluerobotics/ping-python/blob/master/tools/pingproxy.py#L95). When Ping Viewer is in the Device Manager window there are regular writes to multiple different clients, and occasional "general request" messages sent to the device which is followed by several more responses (seemingly one more each time?). When the device is clicked on all communication stops.

```
--------------------------------------------------
ID: 6 - general_request
Header: start_1: 66 start_2: 82 payload_length: 2 message_id: 6 src_device_id: 0 dst_device_id: 0
Payload:
  - requested_id: 4
Checksum: 160 check: 160 pass: True
writing to client ('192.168.2.1', 58410)
writing to client ('192.168.2.1', 51559)
writing to client ('127.0.0.1', 55710)


--------------------------------------------------
ID: 6 - general_request
Header: start_1: 66 start_2: 82 payload_length: 2 message_id: 6 src_device_id: 0 dst_device_id: 0
Payload:
  - requested_id: 4
Checksum: 160 check: 160 pass: True
writing to client ('192.168.2.1', 60037)
writing to client ('192.168.2.1', 58410)
writing to client ('192.168.2.1', 51559)
writing to client ('127.0.0.1', 55710)


--------------------------------------------------
ID: 6 - general_request
Header: start_1: 66 start_2: 82 payload_length: 2 message_id: 6 src_device_id: 0 dst_device_id: 0
Payload:
  - requested_id: 4
Checksum: 160 check: 160 pass: True
writing to client ('192.168.2.1', 60037)
writing to client ('192.168.2.1', 55748)
writing to client ('127.0.0.1', 55710)
writing to client ('192.168.2.1', 58410)
writing to client ('192.168.2.1', 51559)
writing to client ('192.168.2.1', 50870)


--------------------------------------------------
ID: 6 - general_request
Header: start_1: 66 start_2: 82 payload_length: 2 message_id: 6 src_device_id: 0 dst_device_id: 0
Payload:
  - requested_id: 4
Checksum: 160 check: 160 pass: True
writing to client ('192.168.2.1', 60037)
writing to client ('192.168.2.1', 55748)
writing to client ('127.0.0.1', 55710)
writing to client ('192.168.2.1', 64176)
writing to client ('192.168.2.1', 58410)
writing to client ('192.168.2.1', 51559)
writing to client ('192.168.2.1', 50870)


--------------------------------------------------
ID: 6 - general_request
Header: start_1: 66 start_2: 82 payload_length: 2 message_id: 6 src_device_id: 0 dst_device_id: 0
Payload:
  - requested_id: 4
Checksum: 160 check: 160 pass: True
writing to client ('192.168.2.1', 60037)
writing to client ('192.168.2.1', 55445)
writing to client ('192.168.2.1', 55748)
writing to client ('127.0.0.1', 55710)
writing to client ('192.168.2.1', 64176)
writing to client ('192.168.2.1', 58410)
writing to client ('192.168.2.1', 51559)
writing to client ('192.168.2.1', 50870)
```

**Comment by patrickelectric** [comment]
Fixed in #117
